### DISCUSSION TITLE
Where/how to submit documentation issues?

### DISCUSSION THREAD

**Discussion by andres-swax** [OfficeDev/microsoft-365-agents-toolkit#11701] [General]
I found some issue with the documentation, specifically launching a 3-d party tunnel. That's not a bug but a doc deficiency I guess. What's the protocol to report that?   

Specific item is [https://github.com/OfficeDev/teams-toolkit/wiki/%7BDebug%7D-Teams-Toolkit-v5-VS-Code-Tasks#choose-your-own-tunnel-solution](url) issue being that the json for ngrok needs adjustment in order to work.

**Comment by therealjohn** [comment]
> I found some issue with the documentation, specifically launching a 3-d party tunnel. That's not a bug but a doc deficiency I guess. What's the protocol to report that?
> 
> Specific item is [https://github.com/OfficeDev/teams-toolkit/wiki/%7BDebug%7D-Teams-Toolkit-v5-VS-Code-Tasks#choose-your-own-tunnel-solution](url) issue being that the json for ngrok needs adjustment in order to work.

An issue in the repo is fine - what's the adjustment that's needed? Thank you!

**Reply by andres-swax** [reply]
Sorry about the delay replying. My issue was in VS Code Windows.

Change I made to `tasks.json`. The `-Wait` parameter being very important or risk a certain hung/reach count 10/timeout startup situation.

> {
>     "label": "Start local tunnel", ✔
>     "type": "shell", ✔
>     "command": "`Start-Process -FilePath './ngrok' -ArgumentList 'ngrok http 3978 --log=stdout --log-format=logfmt' -NoNewWindow  -Wait`",
> ....
> }

Something else of note is that I had selected Git Bash as the `Terminal > Integrated > Default Profile` on VS Code settings.  This had the effect that the pre-launch task `Start Local Tunnel` which defined as `type:shell` would fail because it expected to be running on Powershell (I think).  I solved this by returning the `Terminal > Integrated > Default Profile` to `null`.

**Comment by XiaofuHuang** [comment]
Thank you for reporting the issue. We will revise the documents in accordance with the latest version of ngrok.

**Reply by andres-swax** [reply]
...it was more on the way the ngrok process is started. See my reply to the other post. 

I think in part the issue is that it is not clear what type of shell the `"type":"shell"` expects.  Hope this helps.
